.circleci/config.yml

```diff
@@ -390,6 +391,41 @@ jobs:
             - target
           key: cargocache-cw20-escrow-rust:1.50.0-{{ checksum "~/project/Cargo.lock" }}
 
+  contract_cw20_ics20:
+    docker:
+      - image: rust:1.50.0
+    working_directory: ~/project/contracts/cw20-ics20
+    steps:
+      - checkout:
+          path: ~/project
+      - run:
+          name: Version information
+          command: rustc --version; cargo --version; rustup --version
+      - restore_cache:
+          keys:
+            - cargocache-cw20-ics20-rust:1.50.0-{{ checksum "~/project/Cargo.lock" }}
+      - run:
+          name: Unit Tests
+          env: RUST_BACKTRACE=1
```

Comment:
This does not work. It must be

```
    environment:
      RUST_BACKTRACE: 1
```

See https://circleci.com/docs/2.0/configuration-reference/

The `env` key is copied over from GotHub Action long time ago and does nothing here.